src/main/java/seedu/jarvis/model/financetracker/InstallmentList.java

```diff
@@ -0,0 +1,92 @@
+package seedu.jarvis.model.financetracker;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+/**
+ * Manages a list of instalments saved by the user.
+ */
+public class InstallmentList {
+    private ArrayList<Installment> allInstallments;
+    private double totalMoneySpentOnInstallments = 0;
```

Comment:
Can we move the initialization of `totalMoneySpentOnInstallments` to the nullary constructor for clarity, since there is a difference in its value on construction between the multiple constructors.

Reply:
noted with thanks!